lib/commands/general.js

```diff
@@ -42,6 +42,18 @@ commands.mobileIsToastVisible = async function (opts = {}) {
   });
 };
 
+commands.mobileRequestPermissions = async function (opts = {}) {
```

Comment:
why make this a mobile: method? i'd prefer to use the `permissions` cap we're already using for ios. or if we want permissions to be granted mid-run, use the appium settings API.

Reply:
It probably makes sense to grant them in runtime as well, but the method should be more universal, so we can do both - grant and revoke a particular permission.

Reply:
I couldn't find an API method that revokes permissions.

Reply:
we can still do it via adb. androidx also contains helpers to execute shell commands